src/runtime_src/core/pcie/driver/linux/xocl/subdev/ert_user.c

```diff
@@ -1308,8 +1308,11 @@ static void ert_user_submit(struct kds_ert *ert, struct kds_command *xcmd)
 	struct xocl_ert_user *ert_user = container_of(ert, struct xocl_ert_user, ert);
 	struct ert_user_command *ecmd = ert_user_alloc_cmd(xcmd);
 
-	if (!ecmd)
+	if (!ecmd) {
+		xcmd->cb.notify_host(xcmd, KDS_ERROR);
```

Comment:
It's very interesting to see if we really hit this code path or not, if we ever get another log from vendor in the future. Can we use a special error code here or print out something?

Reply:
I can print out something.